TypeScript.py

```diff
@@ -8,6 +9,29 @@
 import sublime
 import sublime_plugin
 
+''' Enable logging '''
+logFileLevel = logging.DEBUG
```

Comment:
You might want to make the file log level lower by default (maybe logging.WARN), else if using Sublime Text for long TypeScript sessions without quitting, this might end up being quite a large file.